terraform/modules/enclave/paas-config/prometheus.conf.tpl

```diff
@@ -5,7 +5,7 @@ alerting:
   alertmanagers:
   - scheme: http
     static_configs:
-      - targets: ["${alertmanager_dns_names}"]
+      - targets: ["${alertmanager_dns_names}", "alertman.cluster.re-managed-observe-staging.aws.ext.govsvc.uk"]
```

Comment:
this is in a block with `scheme: http` so i don't think it'll work.
you might need a separate block with `scheme: https`?